q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -95,7 +95,7 @@ def _2(data: Pipeline) -> TaxonomicClassiferTemporaryPickleDirFmt:
 
 
 @plugin.register_transformer
-def _3(dirfmt: TaxonomicClassifierDirFmt) -> dict:
+def _3(dirfmt: TaxonomicClassifierDirFmt) -> Pipeline:
```

Comment:
Since the method now expects a `Pipeline` instead of a `dict`, we need a transformer from the old (deprecated) format into `Pipeline`. That way if a user tries running the method with a .qza in the old format, they'll get a `ValueError`. If we left the return type as `dict`, that transformer wouldn't get invoked and the user would instead get an error stating that the old format doesn't have a transformer into `Pipeline`. With this change to `Pipeline`, users will get a more informative error message about the deprecated format instead of a missing transformer.